hedera-node/hedera-app/src/main/java/com/hedera/node/app/state/merkle/disk/OnDiskValue.java

```diff
@@ -115,9 +117,12 @@ public void serialize(@NonNull final SerializableDataOutputStream out) throws IO
     /** {@inheritDoc} */
     @Override
     public void deserialize(@NonNull final ByteBuffer byteBuffer, int ignored) throws IOException {
+        throw new UnsupportedOperationException("Should never be called");
+        /*
```

Comment:
Probably should remove the commented-out code.

Reply:
OK